tests_e2e/orchestrator/scripts/unzip.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
```

Comment:
we were using the "unzip" package to do this, but that package may not be available in all tested distros (e.g. Flatcar does not even have a package manager) so now we use this Python script to unzip.